client/src/components/Workflow/Editor/Attributes.test.js

```diff
@@ -10,15 +10,15 @@ const TEST_NAME = "workflow_name";
 describe("Attributes", () => {
     it("test attributes", async () => {
```

Comment:
I think we really need to start being more careful describing our tests. The tests are what the next victim uses to fix or modify this code and when it's poorly described, or in this case, not described at all, there is no indication of what it means to pass or fail a test.

It turns the tests into an obstacle instead of a guide.

Reply:
Awesome. That short description of what you're trying to test will help the next person. Are the descriptive tests that john had in the Lint.test.js file no longer valid because of your redesign?

It looks like John had some pretty descriptive tests he added in that file a month ago, maybe you need a rebase to catch them? Or are they no longer relevant after your refactor?